src/templates/search.html

```diff
@@ -11,7 +14,7 @@ <h1 class="search-results-header search-results-header-mobile">
       <h1 class="search-results-header search-results-header-desktop">
         {{ _plural('<b>"{search_query}"</b> returned {n} result',
                    '<b>"{search_query}"</b> returned {n} results',
-                   search_query=query|escape,
+                   search_query=query|safe,
```

Comment:
@mstriemer with escape the code changes made at line 5 & 6 were not working. Can you please advice anything regarding this if we keep it escape itself